cmd/crc/cmd/setup.go

```diff
@@ -31,6 +34,13 @@ var setupCmd = &cobra.Command{
 }
 
 func runSetup(arguments []string) error {
+	if !config.Get(cmdConfig.ConsentTelemetry).AsBool() && terminal.IsTerminal(int(os.Stderr.Fd())) && !globalForce {
+		if input.PromptUserForYesOrNo("Would you like to contribute anonymous usage statistics", false) {
```

Comment:
I would also provide https://developers.redhat.com/article/tool-data-collection in case user want to know about our policy.

Reply:
Ideally we'd be able to differentiate between "ConsentTelemetry was never set", and "ConsentTelemetry has been set to false in a previous crc setup run", at the moment I think we'll be asking again and again if the user answered yes the first time around?

Reply:
@praveenkumar @gbraad I added the privacy notice. 

```
$ crc setup 
CodeReady Containers is constantly improving but we need your feedback.
Would you like to contribute anonymous usage statistics (more details at https://developers.redhat.com/article/tool-data-collection)? [y/N]: y
Thanks for helping us! You can disable telemetry at any time with the command 'crc config set consent-telemetry false'.
INFO Checking if podman remote executable is cached 
INFO Checking if admin-helper executable is cached 
...
```

@cfergeau yes you're totally right. We have no mechanism to detect that right now. I would say that the question is not invasive. If you press directly enter, it assumes false.

Reply:
We can document that if the user doesn't want to see the prompt they can use the `-f` flag which will suppress it

Reply:
>  We have no mechanism to detect that right now. I would say that the question is not invasive. If you press directly enter, it assumes false.

This could be perceived as nagging the user until they say yes, as when they accept, we never ask again, when they refuse, we ask every time crc setup is run.